# robotHeadMotionData.msg
# 
# - joint_data[0] : yaw,   [-30, 30]
# - joint_data[1] : pitch, [-25, 25]
#
# - [-8.000000, 0.000000]

float64[] joint_data
